cli/defenseclaw/tui/services/v8_event_history.py

```
@@ -88,39 +304,221 @@ def __init__(self, store: object) -> None:
+                   ORDER BY timestamp DESC, audit_events.rowid DESC""",
+                (
+                    bounded,
+                    bounded,
+                    bounded,
+                    _MAX_PAYLOAD_BYTES,
+                    _MAX_PAYLOAD_BYTES,
+                    _MAX_FINDING_TAGS_BYTES,
+                ),
+            ).fetchall()
+            return _decode_v8_event_history_rows(rows)
+        else:
```

Ruff (0.16.1)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)